ethdb/database.go

```diff
@@ -37,6 +37,13 @@ type KeyValueWriter interface {
 	Delete(key []byte) error
 }
 
+// RangeDeleter wraps the DeleteRange method of a backing data store.
+type RangeDeleter interface {
```

Comment:
What about KeyValueRangeDeleter? We have `KeyValueReader`, `KeyValueWriter`, and try to keep consistent with others.

Nitpick

Reply:
done